e2e/wdio/driver/wdio.ts

```diff
@@ -290,3 +311,14 @@ export function clickAndDragElement(locationX: number, locationY: number, newLoc
         ]
     }]);
 }
+
+export function selectOptionByAttribute(selector: string, attribute: string, attributeValue: string, index: number = 0): void {
+    click(selector, index);
+    waitForElDisplayed(`${selector} option[${attribute}="${attributeValue}"]`);
+    $(`${selector} option[${attribute}="${attributeValue}"]`).click();
+    pause(3000);
```

Comment:
Anyway to reduce this or use another wait?

Reply:
please, we use `pause(time)` only if we did an action which has expected time to run, like animation